cmd/ocsp-updater/main.go

```diff
@@ -219,42 +333,51 @@ func main() {
 
 		cac, closeChan := setupClients(c, stats)
 
+		updater, err := newUpdater(
+			auditlogger,
+			stats,
+			clock.Default(),
+			dbMap,
+			cac,
+			// Necessary evil for now
+			c.OCSPUpdater,
+		)
+		cmd.FailOnError(err, "Failed to create updater")
+
+		auditlogger.Info(app.VersionString())
```

Comment:
move this up to the top of Action func. There's a ticket open to do that for all of our services, so nice to get it done here now.